feature/platform/transceiver/optics_power_and_bias_current_test/README.md

```diff
@@ -0,0 +1,30 @@
+# gNMI-1.13: Telemetry: Optics Power and Bias Current
```

Comment:
Let's move it to: `feature/platform/tests/optics_power_and_bias_current_test/README.md`?

The `*/tests/*` is required to tell apart functional tests from config gen library unit tests. I removed "transceiver" to shorten the path. Let me know what you think.

Reply:
Close this PR and create a new one under tests dir
  - https://github.com/openconfig/featureprofiles/pull/453